stores/migrations.go

```diff
@@ -1154,3 +1160,19 @@ func performMigration00025_contractState(txn *gorm.DB, logger *zap.SugaredLogger
 	logger.Info("migration 00025_contractState complete")
 	return nil
 }
+
+func performMigration00026_healthValidUntilColumn(txn *gorm.DB, logger *zap.SugaredLogger) error {
+	logger.Info("performing migration 00026_healthValidUntilColumn")
+	if !txn.Migrator().HasColumn(&dbSlab{}, "health_valid_until") {
+		if err := txn.Migrator().AddColumn(&dbSlab{}, "health_valid_until"); err != nil {
```

Comment:
Reminder to manually verify the migration and make sure that the index is actually created.